CafeTime/CafeTime/GlobalSaver.swift

```diff
@@ -13,20 +13,19 @@ class GlobalSaver {
     
     static let shared = GlobalSaver()
     
-    func saveUser(email: String, password: String) {
-        KeychainWrapper.standard.set(email, forKey: Globals.savedEmail)
-        KeychainWrapper.standard.set(password, forKey: Globals.savedPassword)
+    func saveLocalUserCredentials(email: String, password: String) {
+        KeychainWrapper.standard.set(email, forKey: GlobalStrings.savedEmail)
+        KeychainWrapper.standard.set(password, forKey: GlobalStrings.savedPassword)
     }
     
-    func getUserCredentials() -> (email : String?, password: String?) {
-        let email = KeychainWrapper.standard.string(forKey: Globals.savedEmail)
-        let password = KeychainWrapper.standard.string(forKey: Globals.savedPassword)
+    func getLocalUserCredentials() -> (email : String?, password: String?) {
```

Comment:
Probably, we can add some CurrentContext file, where we will keep current user, so we can get it anywhere where we need. And save this info in local database - Realm for example, not in keychain